tools/Customization/DevHome.FileExplorerSourceControlIntegration/Services/RepoStoreOptions.cs

```diff
@@ -9,6 +9,7 @@
 
 namespace DevHome.FileExplorerSourceControlIntegration.Services;
 
+#nullable enable

```

Comment:
What about setting `<Nullable>enable</Nullable>` like we do in the GitIntegration csproj?